src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -5123,14 +5122,22 @@ void ExportMusicXml::textLine(TextLineBase const* const tl, staff_idx_t staff, c
         type = "stop";
     }
 
-    if (hook) {
-        if (hookHeight < 0.0) {
-            lineEnd = "up";
-            hookHeight *= -1.0;
-        } else {
-            lineEnd = "down";
-        }
-        rest += QString(" end-length=\"%1\"").arg(hookHeight * 10);
+    QString lineEnd;
+    switch (hookType) {
+    case HookType::HOOK_90T:
+        lineEnd = "both";
+        rest += QString(" end-length=\"%1\"").arg(fabsf(hookHeight * 20));
```

Comment:
Why `fabsf()` for a `double` rather than `fabs()`

Reply:
And why not even just `std::abs`?
@Jojo-Schmitz It would be great if you could add that to #19533

Reply:
OK, will do

Reply:
done